data-sets-api-server/src/main/java/org/zowe/unix/files/exceptions/NotAEmptyDirectoryException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2019
+ */
+package org.zowe.unix.files.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.zowe.api.common.exceptions.ZoweApiRestException;
+
+public class NotAEmptyDirectoryException extends ZoweApiRestException {
```

Comment:
Should be NotAnEmptyDirectory instead of NotAEmptyDirectory

Reply:
modified to "Not**An**EmptyDirectory "